src/aosm/azext_aosm/tests/latest/unit_test/test_inputs/test_arm_template_input.py

```diff
@@ -82,12 +68,20 @@ def test_get_schema_with_params(self):
             'required': [],
             'type': 'object'
         }
+        print("SCHEMA", schema)
         self.assertEqual(schema, expected_schema)
 
     @patch("builtins.open", mock_open(
         read_data='{"$schema": "#", "resources": { } }'))
```

Comment:
what about this patch open? We should be able to solve it the same was as the previous one